### DISCUSSION TITLE
Extensions Update Checker: Update all button?

### DISCUSSION THREAD

**Discussion by Massimo-B** [xiaoxiaoflood/firefox-scripts#276] [General Support]
Hi, when using the Extensions Update Checker I get informed about new available updates. Then I'm able to update every single Addon.
Is there a way to update all Addons in one go? I like the list of available update before starting the updates automatically, but then like to do all in one go.

**Comment by megamorphg** [comment]
The way the script is designed it basically blocks that action even when you trigger it manually... I ended up disabling the script and subscribing to githubs of the extensions themselves for update descriptions.

**Reply by Massimo-B** [reply]
Thanks. How do you subscribe to extensions on github, can you provide an example? You just open a .xpi URL from the Assets like in https://github.com/onemen/TabMixPlus/releases ? Will that get auto-updates for every such addoon or do you need to browse all the github projects manually for new updates?

**Reply by megamorphg** [reply]
Yes, you can Watch certain things on the repository:
![image](https://github.com/xiaoxiaoflood/firefox-scripts/assets/1782353/d7d81a25-62d9-4da3-8983-ce86b650c9fa)
